shell/android-studio/reicast/src/main/java/com/reicast/emulator/config/OptionsFragment.java

```diff
@@ -411,47 +411,26 @@ public boolean onEditorAction(TextView v, int actionId, KeyEvent event) {
 		});
 
 		int resolutionVertical = mPrefs.getInt(Emulator.pref_resolutionv, Emulator.resolutionv);
-		final String verticalText = getActivity().getString(R.string.resolution_vertical) + " ";
+		final String resolutionText = getActivity().getString(R.string.resolution) + ": ";
 
-		final TextView resolutionVerticalView = getView().findViewById(R.id.resolutionv);
-		resolutionVerticalView.setText((verticalText + "(" + String.valueOf(resolutionVertical) + "%)"));
+		final TextView resolutionTextView = getView().findViewById(R.id.resolution);
+		resolutionTextView.setText((resolutionText + String.valueOf(resolutionVertical) + "%"));
 
-		final SeekBar resolutionVerticalSeek = getView().findViewById(R.id.resolutionv_seekbar);
-		resolutionVerticalSeek.setProgress(resolutionVertical - 1);
-		resolutionVerticalSeek.setIndeterminate(false);
-		resolutionVerticalSeek.setOnSeekBarChangeListener(new OnSeekBarChangeListener() {
+		final SeekBar resolutionVHSeek = getView().findViewById(R.id.resolutionvh_seekbar);
+		resolutionVHSeek.setProgress(resolutionVertical - 1); //can take vertical OR horizontal (the same values)
+		resolutionVHSeek.setIndeterminate(false);
+		resolutionVHSeek.setOnSeekBarChangeListener(new OnSeekBarChangeListener() {
 			public void onProgressChanged(SeekBar seekBar, int progress, boolean fromUser) {
-				resolutionVerticalView.setText((verticalText + "(" + String.valueOf(progress + 1) + "%)"));
+				resolutionTextView.setText((resolutionText + String.valueOf(progress + 1) + "%"));
 			}
 
 			public void onStartTrackingTouch(SeekBar seekBar) {
 			}
 
 			public void onStopTrackingTouch(SeekBar seekBar) {
 				int progress = seekBar.getProgress();
+				//the same progress for both resolutions
 				mPrefs.edit().putInt(Emulator.pref_resolutionv, progress + 1).apply();
```

Comment:
As we now have only one setting (on Android) we should also only have one setting in the ``emu.cfg`` as well.